src/content/wrapped-ether/index.md

```diff
@@ -0,0 +1,75 @@
+
+## Wrapped ether (WETH) vs ether (ETH): What is the difference? {#weth-vs-eth-differences}
+
+Wrapped ether (WETH) and ether (ETH) have subtle differences you should be aware of. For example, the Ethereum protocol recognizes ether—but it has no native knowledge of WETH (and ERC-20 tokens), causing both assets to behave differently. Below are some important distinctions between ETH and WETH tokens below:
+
+|           | **Ether (ETH)**                                                                                                                                                                              | **Wrapped ether (WETH)**                                                                                                                                                                                                                                                                               |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Supply    | **The supply of ETH is managed by the protocol.** The supply of ETH increases when Ethereum validators earn rewards for processing transactions and creating blocks.                         | **WETH is an ERC-20 token whose supply is managed by a smart contract.** New units of WETH are issued by the contract after it receives ETH deposits from users.                                                                                                                                       |
+| Ownership | **Ownership of ETH is defined and guaranteed by the Ethereum blockchain.** For example, protocol rules ensure you can always spend from an account’s ETH balance if you control the account. | **Ownership of WETH depends on the WETH token contract.** The contract defines ownership of WETH tokens and determines the conditions under which you can access your WETH balance.                                                                                                                    |
+| Transfer  | The transfer of ETH is intrinsic to the Ethereum protocol. Transfers are recorded on-chain and publicly verifiable.                                                                          | **The transfer of WETH tokens occurs inside the WETH token contract and [isn’t directly recorded on-chain](https://coinmarketcap.com/alexandria/glossary/internal-transaction).** The blockchain only records the transaction authorizing the smart contract to deduct from the sender’s WETH balance. |
```

Comment:
```suggestion
| Transfer  | The transfer of ETH is intrinsic to the Ethereum protocol. Transfers are recorded on-chain and publicly verifiable.                                                                          | **The transfer of WETH tokens occurs inside the WETH token contract and [isn’t directly recorded on-chain](https://coinmarketcap.com/alexandria/glossary/internal-transaction).** The blockchain only records the transaction authorizing the smart contract to deduct from the sender’s WETH balance. |
```
same as above, dont see any practical reason why average user needs to know this. Unless i am missing something?